docs/content/users/install/shell-completion.md

```diff
@@ -16,7 +16,7 @@ Shells like bash and zsh need help to do this though, they have to know what the
     ```
 
     !!!note "Bash profile"
-        You must add the include to your `.bash_profile` or `.profile` or nothing will work. Use `source ~/.bash_profile` or `source ~/.profile` to make it take effect immediately.
+        You must add the include to your `.bash_profile` or `.profile` or nothing will work. Use `source ~/.bash_profile` or `source ~/.profile` to make it take effect immediately. You will also need to reload your shell either by reopening your terminal window or executing `$ exec zsh`
```

Comment:
I don't believe that's true at all. It's certainly not true with bash. All you have to do is source the profile.